presto-docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -116,6 +116,99 @@ Use exponential backoff starting at 1s up to the value specified by this configu
 
 This property is optional; the default is ``10s``.
 
+Search Guard Authentication
+---------------------------
+
+The Elasticsearch connector provides additional security options to support Elasticsearch clusters that have been configured to use Search Guard.
+
+You can choose certificate format by setting the ``searchguard.ssl.transport.certificate_format`` in the Elasticsearch catalog properties file. This property must be one of the following values:
+
+========================== ========================================================
+Property Value	           Description
+========================== ========================================================
+``NONE`` (default value)   Do not use Search Guard Authentication
+``PEM``                    Use X.509 PEM certificates and PKCS #8 keys 
+``JKS``                    Use Keystore and Truststore files
```

Comment:
Please put `.` at the end of the descriptions as we do in other parts of the doc.